tests/test_objective_funs.py

```diff
@@ -64,3 +66,17 @@ def test_qs_tp(self):
         obj = QuasisymmetryTripleProduct(eq=eq)
         ft = obj.compute(eq.R_lmn, eq.Z_lmn, eq.L_lmn, eq.i_l, eq.Psi)
         np.testing.assert_allclose(ft, 0)
+
+    def test_mercier_stability(self):
+        eq = Equilibrium()
+        obj = MercierStability(eq=eq)
+        DMerc = obj.compute(eq.R_lmn, eq.Z_lmn, eq.L_lmn, eq.p_l, eq.i_l, eq.Psi)
+        np.testing.assert_allclose(DMerc, 0)
```

Comment:
could also throw in a test that the output is the right shape based on the number of surfaces in the grid used